release-calendar/cloud_build.yaml

```diff
@@ -52,3 +52,6 @@ secrets:
       DB_USERNAME: CiQAvF4DHWDrMTBZvbCxwRpkA1AT9k+fRDNvKVFKFmvb9WsBtRQSLwDYFKiUQ+96PiBOialv1MC1VhgctciMC5T5xkOJtURbURyN/O0SgGpRlHm8SGB+
       DB_PASSWORD: CiQAvF4DHVS/vkFxZQxF9AwHS0ELlfGYVSxPONmtfmVaK4PZx2ISLwDYFKiUJgUhlqEZRGNoPxcr27zigWXiZwibVrpLEcaklecTI48xCGvnmmkjwZkG
       BASIC_AUTH: CiQAvF4DHV54zziFx9vYRkcyjAl5gCLTTGHazTnKcrBD7n50OCwSRADYFKiUCZMdSYBgjTKHey0eMiZ3ou7npFPvsNI8+BzfDThhPhOSF0j0a3ZZRbgH/HqGu4L7Tl6of3Ltx0ZVgfBAEup0
+
+# Overall cloud build timeout
+timeout: 1600s
```

Comment:
Curious, why this very specific number? :octopus:

Reply:
Hahha bc I saw it on Google Cloud documentation somewhere.  Ah here https://cloud.google.com/cloud-build/docs/deploying-builds/deploy-appengine